### ISSUE TITLE
Проблема с запуском проекта

### ISSUE THREAD

**Issue by kassymov17** [SUPERMEN4iG/HookahCRM#1]
Доброго времени суток, не удается запустить ваш проект у себя. Не могли бы вы написать, что мне нужно исправить. Поменять строку подключения в файле web.config и? Ошибки в консоли вылетают, о том что нет подключаемых библиотек, скриптов. Я так понимаю это из-за файла bower.json? Спасибо.

**Comment by kkuvardin** [comment]
Да, именно из-за него) довольно давно писалось и использовалось в роли дипломного проекта, функционал минимален, да и интерфейсная часть на angular 1.
Правьте web.config под ваш mssql сервер >= 2012 версии, таблицы приложение само создаст, api работать будет в любом случае.
bower.json я сам залью в репозиторий, как только дома буду т.к там важны версии библиотек. Писалось давно - сейчас версии новые, по-любому проблемы возникнут

**Comment by kassymov17** [comment]
Спасибо, буду ждать)

**Comment by kkuvardin** [comment]
Закомитил

**Comment by kassymov17** [comment]
Спасибо, успешно запустил с:
